tuned/plugins/plugin_sysctl.py

```diff
@@ -16,7 +16,12 @@
 
 class SysctlPlugin(base.Plugin):
 	"""
-	Plugin for applying custom sysctl options.
+	This plugin is used for applying custom `sysctl` settings and should only be used to change 
+        system settings that are not covered by other plugins available in *Tuned*. If the settings
+        are covered by other *Tuned* plugins, please use those plugins instead.
+        
+        The syntax for this plugin is `name=value`, where _name_ is the same as the name provided by
```

Comment:
Perhaps say `variable=value` as you're referring to the `sysctl` tool unless `name=value` is an established convention in the Tuned docs?  See `man sysctl`.  You can find some good wording  like "modify kernel parameters at runtime" in the manual page too, instead of saying applying custom `sysctl` settings.  Would saying that this modifies kernel parameters be more precise and specific?

Reply:
Good point, Jiri. I will fix that. I was using the old docstring info, but you're right that the `sysctl` documentation does say `variable=value`.